scripts/cli/cli_helper.py

```diff
@@ -16,7 +18,7 @@ def format_source(source: List[str]) -> List[str]:
             source_json: List[str] = json.loads(source[0])
             return source_json
         except json.JSONDecodeError as e:
-            get_log().debug("Decoding Json Failed", source=source, msg=e)
+            get_log().debug("Decoding Json Failed", msg=e)
```

Comment:
should this throw ?

Reply:
https://github.com/linz/topo-imagery/pull/117